doc/design/status_port.md

```diff
@@ -0,0 +1,453 @@
+# StatusPort
+
+## Summary and requirements
+
+The `StatusPort` is an alternative to the publish subscriber communication in `iceoryx_posh`.
+
+The target of the `StatusPort` are use-case with the following properties:
+
+* Data is transferred once or is rarely updated
```

Comment:
Requirements to the data would be:
* data must be self-contained (pointers, indices to structures outside are forbidden - caused by torn read)
* data must be trivially copy-able
* all the other SHM constraints of the data must be satisfied

The reader must be able to handle frankenstein objects or mermaids ;)

Reply:
Added.